src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -233,7 +234,19 @@ export const AzureComputeModalBase = ({
     ])
   }
 
-  //TODO this does not actually compute cost as is, see IA-3348
+  const getComputeCost = () => {
```

Comment:
We might want this to live in `runtime-utils` in a section that is labeled as an azure section.

As such, it should probably be parameterized instead of accessing the state, so that the cost is more accessible to any components. 

Additionally, once parameterized, it will be possible tot write a test using the default value.

Reply:
`runtime-utils` is riotously long and tangled; I will see about moving this helper to a parameterized form in the new `cost-utils`.